lips/lip-6.mdx

```diff
@@ -0,0 +1,140 @@
+```
+
+### Validator Owner to be added/removed from the Validator Set
+Once the Validator Owner or its Validator Operator sets the `config` of the Validator Node and is ready to participate in the consensus protocol, it should ask the Association off-chain to be added to the Validator Set. The Association sends a transaction script `add_validator_and_reconfigure`, once it executes, it triggers the Reconfiguration, other validators connect to the Validator Node and it starts participating in the protocol. Note that the size of the Validator Set and hence the size of the quorum for consensus decisions increases.
+
+Under special circumstances (that includes the member leaving the Association or compliance-related issues), the Association may remove a Validator Owner from the Validator Set. It does so by calling the `remove_validator_and_reconfigure` transaction script, it triggers the Reconfiguration. Note that the size of the Validator Set and hence the size of the quorum decreases.
+
+## Properties of the Validator Set enforced off-chain
+
+The following properties are being enforced via contractual obligations:
+* Each Validator Owner is obliged to notify LN about the decision to change the Validator Operator. LN may disallow the switch when it would violate LN's obligations, such as when a new Validator Operator is deemed untrustworthy, or the change unacceptably concentrates consensus power among too few Validator Operators or Validator Nodes' architectures.
+* If an Validator Operator OP_1 is designated to run Validator Nodes for two Validator Owners V_1 and V_2, the Validator Operator must run two distinct instances of Validator Nodes to increase fault-tolerance of the system.
+* The Validator Operator designated by the Validator Owner (or Validator Owner itself if the `operator_account` is unset) is responsible for running a Validator Node with the config in the Validator Set and is responsible for updating/rotating this config as needed.
+
+## Move Modules
+
+Four move modules are involved in the validator configuration management
+
+* [`ValidatorConfig`](https://github.com/libra/libra/blob/master/language/stdlib/modules/ValidatorConfig.move): defines local consensus configuration for a Validator Node of a Validator Owner.
+* [`ValidatorOperatorConfig`](https://github.com/libra/libra/blob/master/language/stdlib/modules/ValidatorOperatorConfig.move): defines the configuration of the Validator Operator (it only contains a human name).
+* [`LibraSystem`](https://github.com/libra/libra/blob/master/language/stdlib/modules/LibraSystem.move): publishes the Validator Set as a global on-chain config and enforces the rules for modifying it.
+* [`LibraConfig`](https://github.com/libra/libra/blob/master/language/stdlib/modules/LibraConfig.move): a generic module that manages all of the global on-chain configurations, including the Validator Set configurations.
+
+## Transaction Scripts
```

Comment:
do we want to talk about the nonce scripts or admin scripts? move currently doesn't allow setting an auth key to all zeroes when creating an account, so it has to be done with the admin script